gno.land/pkg/gnoweb/frontend/js/theme.ts

```diff
@@ -0,0 +1,82 @@
+class Theme {
```

Comment:
Remove the JS please and let's use `prefers-color-scheme` instead as we want to avoid JS when possible.